policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -14,27 +14,32 @@ use linkerd_policy_controller_core::{
 };
 use linkerd_policy_controller_k8s_api::{
     gateway::{self as k8s_gateway_api, ParentReference},
-    policy as linkerd_k8s_api, ResourceExt, Service,
+    policy as linkerd_k8s_api, Pod, ResourceExt, Service,
 };
 use parking_lot::RwLock;
-use std::{hash::Hash, net::IpAddr, num::NonZeroU16, sync::Arc, time};
+use std::{hash::Hash, net::IpAddr, num::NonZeroU16, str::FromStr, sync::Arc, time};
 use tokio::sync::watch;
 
+pub mod grpc;
+pub mod http;
+pub mod metrics;
+mod unmeshed_network;
+
+use unmeshed_network::UnmeshedNetwork;
```

Comment:
nit: would suggest moving this up into the imports above, and prefixing it `use self::unmeshed_network...` to clarify that it's coming from a submodule.